### PR TITLE
Created static programming for the web page

### PR DESCRIPTION
**Pull request by alipwong**

### CODE REVIEW

File: src/pages/syllabus/year12/web_syllabus.rst [hunk 1]
```diff
@@ -0,0 +1,137 @@
+.. role:: python(code)
+   :language: python
+
+Programming for the web
+====================
```
Comment (sjtrny) [inline src/pages/syllabus/year12/web_syllabus.rst:5]: Underline too short

Comment (sjtrny) [inline src/pages/syllabus/year12/web_syllabus.rst:5]: Underline too short

File: src/pages/syllabus/year12/web_syllabus.rst [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+Data transmission using the web
+--------------------
```
Comment (sjtrny) [inline src/pages/syllabus/year12/web_syllabus.rst:8]: Underline too short

File: src/pages/syllabus/year12/web_syllabus.rst [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+* Investigate the effect of big data on web architecture
+
+   **Including:**
+   
+   * data mining
+   * metadata
+   * streaming service management
+
+Designing web applications
+--------------------
```
Comment (sjtrny) [inline src/pages/syllabus/year12/web_syllabus.rst:58]: Underline too short

**Review by sjtrny — COMMENTED** [review]
If it's just one page it probably doesn't need to have its own TOC page. S

File: src/pages/syllabus/year12/web_syllabus.rst (hunk 3 above)
Comment (alipwong) [reply src/pages/syllabus/year12/web_syllabus.rst]: fixed

File: src/pages/syllabus/year12/web_syllabus.rst (hunk 1 above)
Comment (alipwong) [reply src/pages/syllabus/year12/web_syllabus.rst]: fixed

File: src/pages/syllabus/year12/web_syllabus.rst (hunk 2 above)
Comment (alipwong) [reply src/pages/syllabus/year12/web_syllabus.rst]: fixed

**Comment by alipwong** [comment]
> If it's just one page it probably doesn't need to have its own TOC page.

Deleted the index page

**Comment by sjtrny** [comment]
> Deleted the index page

LGTM

**Pull request merged**